tests/test_RBAC.py

```diff
@@ -174,7 +176,9 @@ def test_compute_request_payload(
 ):
     ddo_sample1 = copy.deepcopy(ddo_sample1_v4)
     ddo = Asset(ddo_sample1)
-    ddo.services.append(Service.from_json(1, get_compute_service(None, None, "0x0")))
+    ddo.services.append(
+        Service.from_json(1, get_compute_service(None, None, "0x0", "0x0"))
```

Comment:
please use checksummable addresses here and above, it will save us trouble in the future and I now wonder why these are not checksummed, this should have been detected.